test/System.IdentityModel.Tokens.Jwt.Tests/JwtSecurityTokenHandlerTests.cs

```diff
@@ -40,6 +40,75 @@ namespace System.IdentityModel.Tokens.Jwt.Tests
 {
     public class JwtSecurityTokenHandlerTests
     {
+        [Theory, MemberData(nameof(VerifyValidatedTokenTheoryData))]
+        public void VerifyValidatedToken(JwtTheoryData theoryData)
```

Comment:
minor comment: Can we extend our current _ValidateToken_ tests and add asserts there?
I'm not sure if we need a new method and a theory data set just to test a new model (validated token is null in case of an exception). We are also adding some other tests here, like expecting IDX10501, which are not the goal of this test.

Reply:
Sure. That would cut out lot of code for a minor validation. 
I was wondering if a separate test case is needed for a new change and so created one.

Right, we don't need IDX10501. It is covered by other tests.